app/models/manageiq/providers/azure/cloud_manager/metrics_capture.rb

```diff
@@ -209,6 +209,11 @@ def get_counters(metrics_conn)
           .list('Microsoft.Compute', 'virtualMachines', target.name, target.resource_group.name)
           .select { |m| m.name.value.in?(COUNTER_NAMES) }
       end
+    rescue ::Azure::Armrest::BadRequestException # Probably means region is not supported
```

Comment:
Is there anything more specific on why it failed in the message that we could check?  something like `raise unless err.message =~ /Region does not support metrics/` or whatever the exception message says.

Reply:
> The error message suggests that it's simply a matter of adjusting the api-version string, but in reality none of them work if the region simply isn't supported.

Oh hm okay

Reply:
@agrare yeah, unfortunately the error message is not friendly.